src/utils/apiErrorHandler.ts

```diff
@@ -0,0 +1,49 @@
+let DisplayTexts: any = {
+    "Invalid username.": "I don't see the player with that username, maybe try again.. or If you're searching using ID, Try type \" id\" after your targetted user id!",
+    "Invalid user ID.": "I don't see the player with that id, maybe try again.. or If you're searching using username, Try type \" username\" after your targetted user s' username!"
+}
+
+export class apiErrorHandler {
+    /**
+     * Check Error
+     * @param response Gets request type from node-fetch
+     * @param data Gets request.json from node-fetch
+     * @returns Dictionary
```

Comment:
Dictionary does not exist in javascript, use proper naming.

Reply:
Most of your mistakes were notified in the past 4 reviews, you still commit them. We have eslint, and prettier for a reason, use the tools.